CONTRIBUTNG.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) and 
```

Comment:
Same as the other PR I think here we should mention the birdhouse-deploy one.  This test framework is meant to test that specific deployment.